### PR TITLE
added command line flask command to import data from csv. added a dir…

### PR DESCRIPTION
**Pull request by kdmundale**
Created a directory for csv files to be imported
changed schema to include name a major columns
created an import-csv function and flask command

### CODE REVIEW

**Review by jczimmerman — APPROVED** [review]
Looks great, only thing is we need to remember to remove the CSV from the project after.

**Comment by atmelaku** [comment]
looks good

File: portal/db.py [hunk 1]
```diff
@@ -85,4 +88,31 @@ def init_app(app):
...
+    with open(data_filepath, "r") as f:
+        with get_db() as con:
+            with con.cursor() as cur:
+                reader = csv.reader(f)
+                next(reader)  # Skip the header row.
+                for row in reader:
+                    cur.execute(
+                        "INSERT INTO users VALUES (%s, %s, %s, %s, %s, %s)",
+                        row
+                    )
```
Comment (zachfedor) [inline portal/db.py:109]: At some point, your `users` table might have more columns than this CSV provides so you should be explicit about what columns you are inserting:
```sql
INSERT INTO users (col1, col2, ...)
VALUES (%s, %s, ...)
```

File: import.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import csv
+import os
+import psycopg2
+import psycopg2.extras
+import click
+from flask import current_app, g
+from flask.cli import with_appcontext
```
Comment (zachfedor) [inline import.py:7]: This file doesn't do anything and was probably committed by accident. Best to remove it.

Comment (zachfedor) [reply import.py:7]: @kdmundale fixed in PR #7

**Pull request merged**